  INFO root [0x00001eb8] [2018-06-20 10:57:54,221] - 
  INFO root [0x00001eb8] [2018-06-20 10:57:54,225] - -----------------------------------------------------------------------------------------


  INFO root [0x00001eb8] [2018-06-20 10:57:54,227] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x00001eb8] [2018-06-20 10:57:54,228] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x00001eb8] [2018-06-20 10:57:54,228] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x00001eb8] [2018-06-20 10:57:54,277] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:3559 )
  INFO root [0x00001eb8] [2018-06-20 10:57:54,278] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:3559, SOMAXCONN=5.
  INFO root [0x00001eb8] [2018-06-20 10:57:54,278] - server-configs:
  INFO root [0x00001eb8] [2018-06-20 10:57:54,278] - 	gameUpdateHertz : 10
  INFO root [0x00001eb8] [2018-06-20 10:57:54,278] - 	defaultAoIRadius : 80
  INFO root [0x00001eb8] [2018-06-20 10:57:54,279] - 	defaultAoIHysteresisArea : 5
  INFO root [0x00001eb8] [2018-06-20 10:57:54,279] - 	entryScriptFile : kbemain
  INFO root [0x00001eb8] [2018-06-20 10:57:54,279] - 	internalAddr : 192.168.31.56:3559
  INFO root [0x00001eb8] [2018-06-20 10:57:54,279] - 	componentID : 8001
  INFO root [0x00001eb8] [2018-06-20 10:57:55,793] - ---- cellapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: D25B4E16028C33EB467B7BE2F3500426. Config: Debug X86. Built: 23:20:13 Jun 15 2018. AppID: 8001. UID: 20076. PID: 8328 ----
  INFO root [0x00001eb8] [2018-06-20 10:57:55,795] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x00001eb8] [2018-06-20 10:57:55,795] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00001eb8] [2018-06-20 10:57:55,796] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00001eb8] [2018-06-20 10:57:55,971] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001eb8] [2018-06-20 10:57:55,972] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00001eb8] [2018-06-20 10:57:55,973] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00001eb8] [2018-06-20 10:57:55,979] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001eb8] [2018-06-20 10:57:55,980] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00001eb8] [2018-06-20 10:57:55,981] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001eb8] [2018-06-20 10:57:55,982] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001eb8] [2018-06-20 10:57:56,062] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00001eb8] [2018-06-20 10:57:56,150] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:57) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00001eb8] [2018-06-20 10:57:56,150] - Script::install(): KBExtra is successfully!
  INFO root [0x00001eb8] [2018-06-20 10:57:56,171] - TelnetServer server is running on port 50000
  INFO root [0x00001eb8] [2018-06-20 10:57:57,237] - ---- cellapp is running ----
 DEBUG root [0x00001eb8] [2018-06-20 10:57:57,238] - Components::process(): Request for the process of identity...
  WARN root [0x00001eb8] [2018-06-20 10:58:03,321] - EntityApp::checkTickPeriod: tick took 6084% (6.08 seconds)!
  INFO root [0x00001eb8] [2018-06-20 10:58:04,334] - Components::findComponents: find logger(1)...
  INFO root [0x00001eb8] [2018-06-20 10:58:04,344] - Components::findComponents: found logger, addr:192.168.31.56:3557
  INFO root [0x00001eb8] [2018-06-20 10:58:04,345] - Components::addComponent: reset orderLog, uid=20076!
  INFO root [0x00001eb8] [2018-06-20 10:58:04,346] - Components::addComponent[logger], uid=20076, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001eb8] [2018-06-20 10:58:04,348] - Components::findComponents: find dbmgr(2)...
  INFO root [0x00001eb8] [2018-06-20 10:58:04,449] - Components::findComponents: found dbmgr, addr:192.168.31.56:3555
  INFO root [0x00001eb8] [2018-06-20 10:58:04,449] - Components::addComponent: reset orderLog, uid=20076!
  INFO root [0x00001eb8] [2018-06-20 10:58:04,449] - Components::addComponent[dbmgr], uid=20076, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001eb8] [2018-06-20 10:58:04,451] - Components::findComponents: found dbmgr, addr:192.168.31.56:3555
  WARN root [0x00001eb8] [2018-06-20 10:58:05,451] - Forwarding logs to logger[192.168.31.56:3557/0/0/0]...
 ERROR root [0x00001eb8] [2018-06-20 10:58:33,720] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00001eb8] [2018-06-20 10:58:33,745] - The following logs sent to logger failed:
  WARN root [0x00001eb8] [2018-06-20 10:58:33,746] - ==> [2018-06-20 10:58:33 719] TCPPacketReceiver::processPendingEvents(192.168.31.56:3557): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001eb8] [2018-06-20 10:58:33,746] - ==> [2018-06-20 10:58:33 721] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x00001eb8] [2018-06-20 10:58:33,845] - TCPPacketReceiver::processPendingEvents(192.168.31.56:3555): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001eb8] [2018-06-20 10:58:33,846] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x00001eb8] [2018-06-20 10:58:33,853] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [0x00001eb8] [2018-06-20 10:58:33,867] - Shutdowner::onShutdown: shutting down
  INFO root [0x00001eb8] [2018-06-20 10:58:33,867] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [0x00001eb8] [2018-06-20 10:58:33,868] - EntityRef::destroyObjPool(): size 0.
 DEBUG root [0x00001eb8] [2018-06-20 10:58:33,868] - Witness::destroyObjPool(): size 0.
 DEBUG root [0x00001eb8] [2018-06-20 10:58:33,892] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001eb8] [2018-06-20 10:58:33,893] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00001eb8] [2018-06-20 10:58:33,893] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00001eb8] [2018-06-20 10:58:33,893] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001eb8] [2018-06-20 10:58:33,894] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00001eb8] [2018-06-20 10:58:33,894] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001eb8] [2018-06-20 10:58:33,894] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001eb8] [2018-06-20 10:58:33,894] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x00001eb8] [2018-06-20 10:58:33,910] - Script::uninstall(): is successfully!
 DEBUG root [0x00001eb8] [2018-06-20 10:58:33,910] - ThreadPool::destroy(): starting size 2.
  WARN root [0x00001eb8] [2018-06-20 10:58:34,210] - ThreadPool::destroy(): waiting for thread(2)[0x1962198,0x1962130,], try=1
  INFO root [0x000023dc] [2018-06-20 10:58:51,613] - 
  INFO root [0x000023dc] [2018-06-20 10:58:51,614] - -----------------------------------------------------------------------------------------


  INFO root [0x000023dc] [2018-06-20 10:58:51,616] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x000023dc] [2018-06-20 10:58:51,617] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x000023dc] [2018-06-20 10:58:51,617] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x000023dc] [2018-06-20 10:58:51,643] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:3597 )
  INFO root [0x000023dc] [2018-06-20 10:58:51,645] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:3597, SOMAXCONN=5.
  INFO root [0x000023dc] [2018-06-20 10:58:51,645] - server-configs:
  INFO root [0x000023dc] [2018-06-20 10:58:51,646] - 	gameUpdateHertz : 10
  INFO root [0x000023dc] [2018-06-20 10:58:51,646] - 	defaultAoIRadius : 80
  INFO root [0x000023dc] [2018-06-20 10:58:51,647] - 	defaultAoIHysteresisArea : 5
  INFO root [0x000023dc] [2018-06-20 10:58:51,647] - 	entryScriptFile : kbemain
  INFO root [0x000023dc] [2018-06-20 10:58:51,647] - 	internalAddr : 192.168.31.56:3597
  INFO root [0x000023dc] [2018-06-20 10:58:51,648] - 	componentID : 8001
  INFO root [0x000023dc] [2018-06-20 10:58:53,180] - ---- cellapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: D25B4E16028C33EB467B7BE2F3500426. Config: Debug X86. Built: 23:20:13 Jun 15 2018. AppID: 8001. UID: 20266. PID: 3108 ----
  INFO root [0x000023dc] [2018-06-20 10:58:53,185] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x000023dc] [2018-06-20 10:58:53,191] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x000023dc] [2018-06-20 10:58:53,197] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x000023dc] [2018-06-20 10:58:53,456] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x000023dc] [2018-06-20 10:58:53,457] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x000023dc] [2018-06-20 10:58:53,458] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x000023dc] [2018-06-20 10:58:53,465] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x000023dc] [2018-06-20 10:58:53,466] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x000023dc] [2018-06-20 10:58:53,466] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x000023dc] [2018-06-20 10:58:53,468] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x000023dc] [2018-06-20 10:58:53,559] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x000023dc] [2018-06-20 10:58:53,654] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:57) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x000023dc] [2018-06-20 10:58:53,654] - Script::install(): KBExtra is successfully!
  INFO root [0x000023dc] [2018-06-20 10:58:53,679] - TelnetServer server is running on port 50000
  INFO root [0x000023dc] [2018-06-20 10:58:55,478] - ---- cellapp is running ----
 DEBUG root [0x000023dc] [2018-06-20 10:58:55,479] - Components::process(): Request for the process of identity...
  WARN root [0x000023dc] [2018-06-20 10:59:01,705] - EntityApp::checkTickPeriod: tick took 6227% (6.23 seconds)!
  INFO root [0x000023dc] [2018-06-20 10:59:02,732] - Components::findComponents: find logger(1)...
  INFO root [0x000023dc] [2018-06-20 10:59:02,747] - Components::findComponents: found logger, addr:192.168.31.56:3601
  INFO root [0x000023dc] [2018-06-20 10:59:02,748] - Components::addComponent: reset orderLog, uid=20266!
  INFO root [0x000023dc] [2018-06-20 10:59:02,748] - Components::addComponent[logger], uid=20266, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x000023dc] [2018-06-20 10:59:02,750] - Components::findComponents: find dbmgr(2)...
  INFO root [0x000023dc] [2018-06-20 10:59:02,851] - Components::findComponents: found dbmgr, addr:192.168.31.56:3595
  INFO root [0x000023dc] [2018-06-20 10:59:02,852] - Components::addComponent: reset orderLog, uid=20266!
  INFO root [0x000023dc] [2018-06-20 10:59:02,852] - Components::addComponent[dbmgr], uid=20266, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x000023dc] [2018-06-20 10:59:02,852] - Components::findComponents: found dbmgr, addr:192.168.31.56:3595
  WARN root [0x000023dc] [2018-06-20 10:59:03,853] - Forwarding logs to logger[192.168.31.56:3601/0/0/0]...
 ERROR root [0x000023dc] [2018-06-20 14:22:26,146] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x000023dc] [2018-06-20 14:22:26,147] - The following logs sent to logger failed:
  WARN root [0x000023dc] [2018-06-20 14:22:26,148] - ==> [2018-06-20 14:22:26 144] TCPPacketReceiver::processPendingEvents(192.168.31.56:3601): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x000023dc] [2018-06-20 14:22:26,149] - ==> [2018-06-20 14:22:26 146] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x000023dc] [2018-06-20 14:22:26,262] - TCPPacketReceiver::processPendingEvents(192.168.31.56:3595): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x000023dc] [2018-06-20 14:22:26,267] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x000023dc] [2018-06-20 14:22:26,279] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [0x000023dc] [2018-06-20 14:22:26,280] - Shutdowner::onShutdown: shutting down
  INFO root [0x000023dc] [2018-06-20 14:22:26,281] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [0x000023dc] [2018-06-20 14:22:26,281] - EntityRef::destroyObjPool(): size 0.
 DEBUG root [0x000023dc] [2018-06-20 14:22:26,282] - Witness::destroyObjPool(): size 0.
 DEBUG root [0x000023dc] [2018-06-20 14:22:26,341] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x000023dc] [2018-06-20 14:22:26,341] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x000023dc] [2018-06-20 14:22:26,342] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x000023dc] [2018-06-20 14:22:26,342] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x000023dc] [2018-06-20 14:22:26,342] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x000023dc] [2018-06-20 14:22:26,343] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x000023dc] [2018-06-20 14:22:26,343] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x000023dc] [2018-06-20 14:22:26,343] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x000023dc] [2018-06-20 14:22:26,364] - Script::uninstall(): is successfully!
 DEBUG root [0x000023dc] [2018-06-20 14:22:26,364] - ThreadPool::destroy(): starting size 2.
  WARN root [0x000023dc] [2018-06-20 14:22:26,665] - ThreadPool::destroy(): waiting for thread(2)[0x3431b8,0x343150,], try=1
  INFO root [0x00001950] [2018-06-20 14:22:41,145] - 
  INFO root [0x00001950] [2018-06-20 14:22:41,146] - -----------------------------------------------------------------------------------------


  INFO root [0x00001950] [2018-06-20 14:22:41,148] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x00001950] [2018-06-20 14:22:41,149] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x00001950] [2018-06-20 14:22:41,149] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x00001950] [2018-06-20 14:22:41,162] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:7663 )
  INFO root [0x00001950] [2018-06-20 14:22:41,163] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:7663, SOMAXCONN=5.
  INFO root [0x00001950] [2018-06-20 14:22:41,163] - server-configs:
  INFO root [0x00001950] [2018-06-20 14:22:41,163] - 	gameUpdateHertz : 10
  INFO root [0x00001950] [2018-06-20 14:22:41,164] - 	defaultAoIRadius : 80
  INFO root [0x00001950] [2018-06-20 14:22:41,164] - 	defaultAoIHysteresisArea : 5
  INFO root [0x00001950] [2018-06-20 14:22:41,164] - 	entryScriptFile : kbemain
  INFO root [0x00001950] [2018-06-20 14:22:41,165] - 	internalAddr : 192.168.31.56:7663
  INFO root [0x00001950] [2018-06-20 14:22:41,165] - 	componentID : 8001
  INFO root [0x00001950] [2018-06-20 14:22:42,679] - ---- cellapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: D25B4E16028C33EB467B7BE2F3500426. Config: Debug X86. Built: 23:20:13 Jun 15 2018. AppID: 8001. UID: 27432. PID: 9960 ----
  INFO root [0x00001950] [2018-06-20 14:22:42,681] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x00001950] [2018-06-20 14:22:42,681] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00001950] [2018-06-20 14:22:42,682] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00001950] [2018-06-20 14:22:42,860] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001950] [2018-06-20 14:22:42,861] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00001950] [2018-06-20 14:22:42,862] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00001950] [2018-06-20 14:22:42,869] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001950] [2018-06-20 14:22:42,870] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00001950] [2018-06-20 14:22:42,870] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001950] [2018-06-20 14:22:42,872] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001950] [2018-06-20 14:22:42,985] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00001950] [2018-06-20 14:22:43,069] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:57) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00001950] [2018-06-20 14:22:43,070] - Script::install(): KBExtra is successfully!
  INFO root [0x00001950] [2018-06-20 14:22:43,092] - TelnetServer server is running on port 50000
  INFO root [0x00001950] [2018-06-20 14:22:44,617] - ---- cellapp is running ----
 DEBUG root [0x00001950] [2018-06-20 14:22:44,618] - Components::process(): Request for the process of identity...
  WARN root [0x00001950] [2018-06-20 14:22:50,727] - EntityApp::checkTickPeriod: tick took 6109% (6.11 seconds)!
  INFO root [0x00001950] [2018-06-20 14:22:51,818] - Components::findComponents: find logger(1)...
  INFO root [0x00001950] [2018-06-20 14:22:51,861] - Components::findComponents: found logger, addr:192.168.31.56:7659
  INFO root [0x00001950] [2018-06-20 14:22:51,861] - Components::addComponent: reset orderLog, uid=27432!
  INFO root [0x00001950] [2018-06-20 14:22:51,862] - Components::addComponent[logger], uid=27432, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001950] [2018-06-20 14:22:51,865] - Components::findComponents: find dbmgr(2)...
  INFO root [0x00001950] [2018-06-20 14:22:51,971] - Components::findComponents: found dbmgr, addr:192.168.31.56:7661
  INFO root [0x00001950] [2018-06-20 14:22:51,971] - Components::addComponent: reset orderLog, uid=27432!
  INFO root [0x00001950] [2018-06-20 14:22:51,971] - Components::addComponent[dbmgr], uid=27432, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001950] [2018-06-20 14:22:51,972] - Components::findComponents: found dbmgr, addr:192.168.31.56:7661
  WARN root [0x00001950] [2018-06-20 14:22:52,972] - EntityApp::checkTickPeriod: tick took 1254% (1.25 seconds)!
  WARN root [0x00001950] [2018-06-20 14:22:52,973] - Forwarding logs to logger[192.168.31.56:7659/0/0/0]...
 ERROR root [0x00001950] [2018-06-20 14:23:01,203] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00001950] [2018-06-20 14:23:01,211] - The following logs sent to logger failed:
  WARN root [0x00001950] [2018-06-20 14:23:01,212] - ==> [2018-06-20 14:23:01 200] TCPPacketReceiver::processPendingEvents(192.168.31.56:7659): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001950] [2018-06-20 14:23:01,213] - ==> [2018-06-20 14:23:01 204] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x00001950] [2018-06-20 14:23:01,334] - TCPPacketReceiver::processPendingEvents(192.168.31.56:7661): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001950] [2018-06-20 14:23:01,336] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x00001950] [2018-06-20 14:23:01,345] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [0x00001950] [2018-06-20 14:23:01,357] - Shutdowner::onShutdown: shutting down
  INFO root [0x00001950] [2018-06-20 14:23:01,357] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [0x00001950] [2018-06-20 14:23:01,358] - EntityRef::destroyObjPool(): size 0.
 DEBUG root [0x00001950] [2018-06-20 14:23:01,359] - Witness::destroyObjPool(): size 0.
 DEBUG root [0x00001950] [2018-06-20 14:23:01,393] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001950] [2018-06-20 14:23:01,394] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00001950] [2018-06-20 14:23:01,394] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00001950] [2018-06-20 14:23:01,394] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001950] [2018-06-20 14:23:01,395] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00001950] [2018-06-20 14:23:01,395] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001950] [2018-06-20 14:23:01,395] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001950] [2018-06-20 14:23:01,396] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x00001950] [2018-06-20 14:23:01,417] - Script::uninstall(): is successfully!
 DEBUG root [0x00001950] [2018-06-20 14:23:01,417] - ThreadPool::destroy(): starting size 2.
  WARN root [0x00001950] [2018-06-20 14:23:01,717] - ThreadPool::destroy(): waiting for thread(2)[0x192198,0x192130,], try=1
  INFO root [0x00002708] [2018-06-20 14:23:18,708] - 
  INFO root [0x00002708] [2018-06-20 14:23:18,711] - -----------------------------------------------------------------------------------------


  INFO root [0x00002708] [2018-06-20 14:23:18,713] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x00002708] [2018-06-20 14:23:18,714] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x00002708] [2018-06-20 14:23:18,715] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x00002708] [2018-06-20 14:23:18,728] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:7699 )
  INFO root [0x00002708] [2018-06-20 14:23:18,728] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:7699, SOMAXCONN=5.
  INFO root [0x00002708] [2018-06-20 14:23:18,728] - server-configs:
  INFO root [0x00002708] [2018-06-20 14:23:18,729] - 	gameUpdateHertz : 10
  INFO root [0x00002708] [2018-06-20 14:23:18,729] - 	defaultAoIRadius : 80
  INFO root [0x00002708] [2018-06-20 14:23:18,729] - 	defaultAoIHysteresisArea : 5
  INFO root [0x00002708] [2018-06-20 14:23:18,730] - 	entryScriptFile : kbemain
  INFO root [0x00002708] [2018-06-20 14:23:18,730] - 	internalAddr : 192.168.31.56:7699
  INFO root [0x00002708] [2018-06-20 14:23:18,730] - 	componentID : 8001
  INFO root [0x00002708] [2018-06-20 14:23:20,240] - ---- cellapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: D25B4E16028C33EB467B7BE2F3500426. Config: Debug X86. Built: 23:20:13 Jun 15 2018. AppID: 8001. UID: 27557. PID: 2928 ----
  INFO root [0x00002708] [2018-06-20 14:23:20,241] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x00002708] [2018-06-20 14:23:20,241] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00002708] [2018-06-20 14:23:20,241] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00002708] [2018-06-20 14:23:20,506] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00002708] [2018-06-20 14:23:20,507] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00002708] [2018-06-20 14:23:20,508] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00002708] [2018-06-20 14:23:20,516] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00002708] [2018-06-20 14:23:20,517] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00002708] [2018-06-20 14:23:20,517] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00002708] [2018-06-20 14:23:20,519] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00002708] [2018-06-20 14:23:20,649] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00002708] [2018-06-20 14:23:20,738] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:57) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00002708] [2018-06-20 14:23:20,739] - Script::install(): KBExtra is successfully!
  INFO root [0x00002708] [2018-06-20 14:23:20,762] - TelnetServer server is running on port 50000
  INFO root [0x00002708] [2018-06-20 14:23:22,206] - ---- cellapp is running ----
 DEBUG root [0x00002708] [2018-06-20 14:23:22,207] - Components::process(): Request for the process of identity...
  WARN root [0x00002708] [2018-06-20 14:23:28,319] - EntityApp::checkTickPeriod: tick took 6111% (6.11 seconds)!
  INFO root [0x00002708] [2018-06-20 14:23:29,408] - Components::findComponents: find logger(1)...
  INFO root [0x00002708] [2018-06-20 14:23:29,499] - Components::findComponents: found logger, addr:192.168.31.56:7693
  INFO root [0x00002708] [2018-06-20 14:23:29,499] - Components::addComponent: reset orderLog, uid=27557!
  INFO root [0x00002708] [2018-06-20 14:23:29,499] - Components::addComponent[logger], uid=27557, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00002708] [2018-06-20 14:23:29,501] - Components::findComponents: find dbmgr(2)...
  INFO root [0x00002708] [2018-06-20 14:23:29,602] - Components::findComponents: found dbmgr, addr:192.168.31.56:7696
  INFO root [0x00002708] [2018-06-20 14:23:29,603] - Components::addComponent: reset orderLog, uid=27557!
  INFO root [0x00002708] [2018-06-20 14:23:29,603] - Components::addComponent[dbmgr], uid=27557, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00002708] [2018-06-20 14:23:29,604] - Components::findComponents: found dbmgr, addr:192.168.31.56:7696
  WARN root [0x00002708] [2018-06-20 14:23:30,604] - EntityApp::checkTickPeriod: tick took 1296% (1.30 seconds)!
  WARN root [0x00002708] [2018-06-20 14:23:30,605] - Forwarding logs to logger[192.168.31.56:7693/0/0/0]...
 ERROR root [0x00002708] [2018-06-20 19:26:38,832] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00002708] [2018-06-20 19:26:38,838] - The following logs sent to logger failed:
  WARN root [0x00002708] [2018-06-20 19:26:38,839] - ==> [2018-06-20 19:26:38 827] TCPPacketReceiver::processPendingEvents(192.168.31.56:7693): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00002708] [2018-06-20 19:26:38,839] - ==> [2018-06-20 19:26:38 834] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x00002708] [2018-06-20 19:26:38,937] - TCPPacketReceiver::processPendingEvents(192.168.31.56:7696): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00002708] [2018-06-20 19:26:38,942] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x00002708] [2018-06-20 19:26:38,955] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [0x00002708] [2018-06-20 19:26:38,957] - Shutdowner::onShutdown: shutting down
  INFO root [0x00002708] [2018-06-20 19:26:38,958] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [0x00002708] [2018-06-20 19:26:38,958] - EntityRef::destroyObjPool(): size 0.
 DEBUG root [0x00002708] [2018-06-20 19:26:38,959] - Witness::destroyObjPool(): size 0.
 DEBUG root [0x00002708] [2018-06-20 19:26:38,984] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00002708] [2018-06-20 19:26:38,984] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00002708] [2018-06-20 19:26:38,985] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00002708] [2018-06-20 19:26:38,985] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00002708] [2018-06-20 19:26:38,985] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00002708] [2018-06-20 19:26:38,985] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00002708] [2018-06-20 19:26:38,985] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00002708] [2018-06-20 19:26:38,986] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x00002708] [2018-06-20 19:26:39,004] - Script::uninstall(): is successfully!
 DEBUG root [0x00002708] [2018-06-20 19:26:39,005] - ThreadPool::destroy(): starting size 2.
  WARN root [0x00002708] [2018-06-20 19:26:39,305] - ThreadPool::destroy(): waiting for thread(2)[0x3b31b8,0x3b3150,], try=1
  INFO root [0x00002a9c] [2018-06-20 19:26:57,262] - 
  INFO root [0x00002a9c] [2018-06-20 19:26:57,283] - -----------------------------------------------------------------------------------------


  INFO root [0x00002a9c] [2018-06-20 19:26:57,285] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x00002a9c] [2018-06-20 19:26:57,285] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x00002a9c] [2018-06-20 19:26:57,286] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x00002a9c] [2018-06-20 19:26:57,300] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:10542 )
  INFO root [0x00002a9c] [2018-06-20 19:26:57,301] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:10542, SOMAXCONN=5.
  INFO root [0x00002a9c] [2018-06-20 19:26:57,301] - server-configs:
  INFO root [0x00002a9c] [2018-06-20 19:26:57,302] - 	gameUpdateHertz : 10
  INFO root [0x00002a9c] [2018-06-20 19:26:57,302] - 	defaultAoIRadius : 80
  INFO root [0x00002a9c] [2018-06-20 19:26:57,302] - 	defaultAoIHysteresisArea : 5
  INFO root [0x00002a9c] [2018-06-20 19:26:57,302] - 	entryScriptFile : kbemain
  INFO root [0x00002a9c] [2018-06-20 19:26:57,303] - 	internalAddr : 192.168.31.56:10542
  INFO root [0x00002a9c] [2018-06-20 19:26:57,303] - 	componentID : 8001
  INFO root [0x00002a9c] [2018-06-20 19:26:58,818] - ---- cellapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: D25B4E16028C33EB467B7BE2F3500426. Config: Debug X86. Built: 23:20:13 Jun 15 2018. AppID: 8001. UID: 21516. PID: 8064 ----
  INFO root [0x00002a9c] [2018-06-20 19:26:58,820] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x00002a9c] [2018-06-20 19:26:58,820] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00002a9c] [2018-06-20 19:26:58,821] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00002a9c] [2018-06-20 19:26:59,128] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00002a9c] [2018-06-20 19:26:59,129] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00002a9c] [2018-06-20 19:26:59,130] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00002a9c] [2018-06-20 19:26:59,178] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00002a9c] [2018-06-20 19:26:59,179] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00002a9c] [2018-06-20 19:26:59,179] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00002a9c] [2018-06-20 19:26:59,181] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00002a9c] [2018-06-20 19:26:59,280] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00002a9c] [2018-06-20 19:26:59,475] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:57) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00002a9c] [2018-06-20 19:26:59,475] - Script::install(): KBExtra is successfully!
  INFO root [0x00002a9c] [2018-06-20 19:26:59,630] - TelnetServer server is running on port 50000
  INFO root [0x00002a9c] [2018-06-20 19:27:01,610] - ---- cellapp is running ----
 DEBUG root [0x00002a9c] [2018-06-20 19:27:01,611] - Components::process(): Request for the process of identity...
  WARN root [0x00002a9c] [2018-06-20 19:27:07,806] - EntityApp::checkTickPeriod: tick took 6195% (6.20 seconds)!
  INFO root [0x00002a9c] [2018-06-20 19:27:08,890] - Components::findComponents: find logger(1)...
  INFO root [0x00002a9c] [2018-06-20 19:27:08,986] - Components::findComponents: found logger, addr:192.168.31.56:10549
  INFO root [0x00002a9c] [2018-06-20 19:27:08,986] - Components::addComponent: reset orderLog, uid=21516!
  INFO root [0x00002a9c] [2018-06-20 19:27:08,986] - Components::addComponent[logger], uid=21516, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00002a9c] [2018-06-20 19:27:08,988] - Components::findComponents: find dbmgr(2)...
  INFO root [0x00002a9c] [2018-06-20 19:27:09,090] - Components::findComponents: found dbmgr, addr:192.168.31.56:10547
  INFO root [0x00002a9c] [2018-06-20 19:27:09,091] - Components::addComponent: reset orderLog, uid=21516!
  INFO root [0x00002a9c] [2018-06-20 19:27:09,091] - Components::addComponent[dbmgr], uid=21516, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00002a9c] [2018-06-20 19:27:09,092] - Components::findComponents: found dbmgr, addr:192.168.31.56:10547
  WARN root [0x00002a9c] [2018-06-20 19:27:10,092] - Forwarding logs to logger[192.168.31.56:10549/0/0/0]...
 ERROR root [0x00002a9c] [2018-06-20 19:29:30,226] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00002a9c] [2018-06-20 19:29:30,241] - The following logs sent to logger failed:
  WARN root [0x00002a9c] [2018-06-20 19:29:30,242] - ==> [2018-06-20 19:29:30 219] TCPPacketReceiver::processPendingEvents(192.168.31.56:10549): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00002a9c] [2018-06-20 19:29:30,243] - ==> [2018-06-20 19:29:30 226] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x00002a9c] [2018-06-20 19:29:30,343] - TCPPacketReceiver::processPendingEvents(192.168.31.56:10547): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00002a9c] [2018-06-20 19:29:30,349] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x00002a9c] [2018-06-20 19:29:30,365] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [0x00002a9c] [2018-06-20 19:29:30,373] - Shutdowner::onShutdown: shutting down
  INFO root [0x00002a9c] [2018-06-20 19:29:30,374] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [0x00002a9c] [2018-06-20 19:29:30,374] - EntityRef::destroyObjPool(): size 0.
 DEBUG root [0x00002a9c] [2018-06-20 19:29:30,375] - Witness::destroyObjPool(): size 0.
 DEBUG root [0x00002a9c] [2018-06-20 19:29:30,408] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00002a9c] [2018-06-20 19:29:30,408] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00002a9c] [2018-06-20 19:29:30,409] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00002a9c] [2018-06-20 19:29:30,409] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00002a9c] [2018-06-20 19:29:30,409] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00002a9c] [2018-06-20 19:29:30,410] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00002a9c] [2018-06-20 19:29:30,410] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00002a9c] [2018-06-20 19:29:30,410] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x00002a9c] [2018-06-20 19:29:30,430] - Script::uninstall(): is successfully!
 DEBUG root [0x00002a9c] [2018-06-20 19:29:30,430] - ThreadPool::destroy(): starting size 2.
  WARN root [0x00002a9c] [2018-06-20 19:29:30,731] - ThreadPool::destroy(): waiting for thread(2)[0x8131b8,0x813150,], try=1
  INFO root [0x00001d24] [2018-06-20 19:29:44,809] - 
  INFO root [0x00001d24] [2018-06-20 19:29:44,810] - -----------------------------------------------------------------------------------------


  INFO root [0x00001d24] [2018-06-20 19:29:44,813] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x00001d24] [2018-06-20 19:29:44,813] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x00001d24] [2018-06-20 19:29:44,814] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x00001d24] [2018-06-20 19:29:44,895] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:10619 )
  INFO root [0x00001d24] [2018-06-20 19:29:44,896] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:10619, SOMAXCONN=5.
  INFO root [0x00001d24] [2018-06-20 19:29:44,896] - server-configs:
  INFO root [0x00001d24] [2018-06-20 19:29:44,896] - 	gameUpdateHertz : 10
  INFO root [0x00001d24] [2018-06-20 19:29:44,897] - 	defaultAoIRadius : 80
  INFO root [0x00001d24] [2018-06-20 19:29:44,897] - 	defaultAoIHysteresisArea : 5
  INFO root [0x00001d24] [2018-06-20 19:29:44,897] - 	entryScriptFile : kbemain
  INFO root [0x00001d24] [2018-06-20 19:29:44,916] - 	internalAddr : 192.168.31.56:10619
  INFO root [0x00001d24] [2018-06-20 19:29:44,916] - 	componentID : 8001
  INFO root [0x00001d24] [2018-06-20 19:29:46,464] - ---- cellapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: D25B4E16028C33EB467B7BE2F3500426. Config: Debug X86. Built: 23:20:13 Jun 15 2018. AppID: 8001. UID: 22062. PID: 7604 ----
  INFO root [0x00001d24] [2018-06-20 19:29:46,465] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x00001d24] [2018-06-20 19:29:46,466] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00001d24] [2018-06-20 19:29:46,467] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00001d24] [2018-06-20 19:29:46,656] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001d24] [2018-06-20 19:29:46,657] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00001d24] [2018-06-20 19:29:46,658] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00001d24] [2018-06-20 19:29:46,698] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001d24] [2018-06-20 19:29:46,699] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00001d24] [2018-06-20 19:29:46,699] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001d24] [2018-06-20 19:29:46,701] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001d24] [2018-06-20 19:29:46,840] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00001d24] [2018-06-20 19:29:47,023] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:57) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00001d24] [2018-06-20 19:29:47,024] - Script::install(): KBExtra is successfully!
  INFO root [0x00001d24] [2018-06-20 19:29:47,050] - TelnetServer server is running on port 50000
  INFO root [0x00001d24] [2018-06-20 19:29:48,250] - ---- cellapp is running ----
 DEBUG root [0x00001d24] [2018-06-20 19:29:48,251] - Components::process(): Request for the process of identity...
  WARN root [0x00001d24] [2018-06-20 19:29:54,384] - EntityApp::checkTickPeriod: tick took 6133% (6.13 seconds)!
  INFO root [0x00001d24] [2018-06-20 19:29:55,416] - Components::findComponents: find logger(1)...
  INFO root [0x00001d24] [2018-06-20 19:29:55,436] - Components::findComponents: found logger, addr:192.168.31.56:10615
  INFO root [0x00001d24] [2018-06-20 19:29:55,437] - Components::addComponent: reset orderLog, uid=22062!
  INFO root [0x00001d24] [2018-06-20 19:29:55,437] - Components::addComponent[logger], uid=22062, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001d24] [2018-06-20 19:29:55,438] - Components::findComponents: find dbmgr(2)...
  INFO root [0x00001d24] [2018-06-20 19:29:55,543] - Components::findComponents: found dbmgr, addr:192.168.31.56:10621
  INFO root [0x00001d24] [2018-06-20 19:29:55,544] - Components::addComponent: reset orderLog, uid=22062!
  INFO root [0x00001d24] [2018-06-20 19:29:55,544] - Components::addComponent[dbmgr], uid=22062, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001d24] [2018-06-20 19:29:55,544] - Components::findComponents: found dbmgr, addr:192.168.31.56:10621
  WARN root [0x00001d24] [2018-06-20 19:29:56,544] - Forwarding logs to logger[192.168.31.56:10615/0/0/0]...
 ERROR root [0x00001d24] [2018-06-20 19:31:24,363] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00001d24] [2018-06-20 19:31:24,384] - The following logs sent to logger failed:
  WARN root [0x00001d24] [2018-06-20 19:31:24,385] - ==> [2018-06-20 19:31:24 359] TCPPacketReceiver::processPendingEvents(192.168.31.56:10615): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001d24] [2018-06-20 19:31:24,385] - ==> [2018-06-20 19:31:24 364] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x00001d24] [2018-06-20 19:31:24,523] - TCPPacketReceiver::processPendingEvents(192.168.31.56:10621): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001d24] [2018-06-20 19:31:24,532] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x00001d24] [2018-06-20 19:31:24,559] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [0x00001d24] [2018-06-20 19:31:24,560] - Shutdowner::onShutdown: shutting down
  INFO root [0x00001d24] [2018-06-20 19:31:24,561] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [0x00001d24] [2018-06-20 19:31:24,561] - EntityRef::destroyObjPool(): size 0.
 DEBUG root [0x00001d24] [2018-06-20 19:31:24,561] - Witness::destroyObjPool(): size 0.
 DEBUG root [0x00001d24] [2018-06-20 19:31:24,596] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001d24] [2018-06-20 19:31:24,596] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00001d24] [2018-06-20 19:31:24,596] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00001d24] [2018-06-20 19:31:24,597] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001d24] [2018-06-20 19:31:24,597] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00001d24] [2018-06-20 19:31:24,597] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001d24] [2018-06-20 19:31:24,598] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001d24] [2018-06-20 19:31:24,598] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x00001d24] [2018-06-20 19:31:24,622] - Script::uninstall(): is successfully!
 DEBUG root [0x00001d24] [2018-06-20 19:31:24,623] - ThreadPool::destroy(): starting size 2.
  WARN root [0x00001d24] [2018-06-20 19:31:24,923] - ThreadPool::destroy(): waiting for thread(2)[0x5131b8,0x513150,], try=1
  INFO root [0x00002238] [2018-06-20 19:32:43,360] - 
  INFO root [0x00002238] [2018-06-20 19:32:43,368] - -----------------------------------------------------------------------------------------


  INFO root [0x00002238] [2018-06-20 19:32:43,371] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x00002238] [2018-06-20 19:32:43,371] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x00002238] [2018-06-20 19:32:43,376] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x00002238] [2018-06-20 19:32:43,388] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:10691 )
  INFO root [0x00002238] [2018-06-20 19:32:43,388] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:10691, SOMAXCONN=5.
  INFO root [0x00002238] [2018-06-20 19:32:43,389] - server-configs:
  INFO root [0x00002238] [2018-06-20 19:32:43,389] - 	gameUpdateHertz : 10
  INFO root [0x00002238] [2018-06-20 19:32:43,390] - 	defaultAoIRadius : 80
  INFO root [0x00002238] [2018-06-20 19:32:43,390] - 	defaultAoIHysteresisArea : 5
  INFO root [0x00002238] [2018-06-20 19:32:43,390] - 	entryScriptFile : kbemain
  INFO root [0x00002238] [2018-06-20 19:32:43,391] - 	internalAddr : 192.168.31.56:10691
  INFO root [0x00002238] [2018-06-20 19:32:43,391] - 	componentID : 8001
  WARN root [0x00002238] [2018-06-20 19:32:43,725] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20716, errorno=10048
  INFO root [0x00002238] [2018-06-20 19:32:43,812] - ---- cellapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: D25B4E16028C33EB467B7BE2F3500426. Config: Debug X86. Built: 23:20:13 Jun 15 2018. AppID: 8001. UID: 22646. PID: 12180 ----
  INFO root [0x00002238] [2018-06-20 19:32:43,813] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x00002238] [2018-06-20 19:32:43,814] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00002238] [2018-06-20 19:32:43,815] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00002238] [2018-06-20 19:32:44,018] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00002238] [2018-06-20 19:32:44,020] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00002238] [2018-06-20 19:32:44,021] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00002238] [2018-06-20 19:32:44,030] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00002238] [2018-06-20 19:32:44,031] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00002238] [2018-06-20 19:32:44,031] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00002238] [2018-06-20 19:32:44,033] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00002238] [2018-06-20 19:32:44,148] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00002238] [2018-06-20 19:32:44,269] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:57) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00002238] [2018-06-20 19:32:44,270] - Script::install(): KBExtra is successfully!
  INFO root [0x00002238] [2018-06-20 19:32:44,300] - TelnetServer server is running on port 50000
  INFO root [0x00002238] [2018-06-20 19:32:47,035] - ---- cellapp is running ----
 DEBUG root [0x00002238] [2018-06-20 19:32:47,038] - Components::process(): Request for the process of identity...
  WARN root [0x00002238] [2018-06-20 19:32:53,404] - EntityApp::checkTickPeriod: tick took 6368% (6.37 seconds)!
  INFO root [0x00002238] [2018-06-20 19:32:54,436] - Components::findComponents: find logger(1)...
  INFO root [0x00002238] [2018-06-20 19:32:54,468] - Components::findComponents: found logger, addr:192.168.31.56:10686
  INFO root [0x00002238] [2018-06-20 19:32:54,468] - Components::addComponent: reset orderLog, uid=22646!
  INFO root [0x00002238] [2018-06-20 19:32:54,468] - Components::addComponent[logger], uid=22646, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00002238] [2018-06-20 19:32:54,470] - Components::findComponents: find dbmgr(2)...
  INFO root [0x00002238] [2018-06-20 19:32:54,571] - Components::findComponents: found dbmgr, addr:192.168.31.56:10692
  INFO root [0x00002238] [2018-06-20 19:32:54,571] - Components::addComponent: reset orderLog, uid=22646!
  INFO root [0x00002238] [2018-06-20 19:32:54,572] - Components::addComponent[dbmgr], uid=22646, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00002238] [2018-06-20 19:32:54,572] - Components::findComponents: found dbmgr, addr:192.168.31.56:10692
  WARN root [0x00002238] [2018-06-20 19:32:55,572] - EntityApp::checkTickPeriod: tick took 1236% (1.24 seconds)!
  WARN root [0x00002238] [2018-06-20 19:32:55,573] - Forwarding logs to logger[192.168.31.56:10686/0/0/0]...
 ERROR root [0x00002238] [2018-06-20 19:58:15,820] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00002238] [2018-06-20 19:58:15,822] - The following logs sent to logger failed:
  WARN root [0x00002238] [2018-06-20 19:58:15,822] - ==> [2018-06-20 19:58:15 813] TCPPacketReceiver::processPendingEvents(192.168.31.56:10686): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00002238] [2018-06-20 19:58:15,823] - ==> [2018-06-20 19:58:15 821] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x00002238] [2018-06-20 19:58:15,927] - TCPPacketReceiver::processPendingEvents(192.168.31.56:10692): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00002238] [2018-06-20 19:58:15,935] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x00002238] [2018-06-20 19:58:15,950] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [0x00002238] [2018-06-20 19:58:15,954] - Shutdowner::onShutdown: shutting down
  INFO root [0x00002238] [2018-06-20 19:58:15,954] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [0x00002238] [2018-06-20 19:58:15,955] - EntityRef::destroyObjPool(): size 0.
 DEBUG root [0x00002238] [2018-06-20 19:58:15,955] - Witness::destroyObjPool(): size 0.
 DEBUG root [0x00002238] [2018-06-20 19:58:15,989] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00002238] [2018-06-20 19:58:15,989] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00002238] [2018-06-20 19:58:15,989] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00002238] [2018-06-20 19:58:15,990] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00002238] [2018-06-20 19:58:15,990] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00002238] [2018-06-20 19:58:15,990] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00002238] [2018-06-20 19:58:15,991] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00002238] [2018-06-20 19:58:15,991] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x00002238] [2018-06-20 19:58:16,010] - Script::uninstall(): is successfully!
 DEBUG root [0x00002238] [2018-06-20 19:58:16,011] - ThreadPool::destroy(): starting size 2.
  WARN root [0x00002238] [2018-06-20 19:58:16,312] - ThreadPool::destroy(): waiting for thread(2)[0x1d73220,0x1d731b8,], try=1
  INFO root [0x0000249c] [2018-06-20 19:58:27,319] - 
  INFO root [0x0000249c] [2018-06-20 19:58:27,320] - -----------------------------------------------------------------------------------------


  INFO root [0x0000249c] [2018-06-20 19:58:27,321] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x0000249c] [2018-06-20 19:58:27,322] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x0000249c] [2018-06-20 19:58:27,322] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x0000249c] [2018-06-20 19:58:27,334] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:10907 )
  INFO root [0x0000249c] [2018-06-20 19:58:27,334] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:10907, SOMAXCONN=5.
  INFO root [0x0000249c] [2018-06-20 19:58:27,335] - server-configs:
  INFO root [0x0000249c] [2018-06-20 19:58:27,335] - 	gameUpdateHertz : 10
  INFO root [0x0000249c] [2018-06-20 19:58:27,335] - 	defaultAoIRadius : 80
  INFO root [0x0000249c] [2018-06-20 19:58:27,335] - 	defaultAoIHysteresisArea : 5
  INFO root [0x0000249c] [2018-06-20 19:58:27,336] - 	entryScriptFile : kbemain
  INFO root [0x0000249c] [2018-06-20 19:58:27,336] - 	internalAddr : 192.168.31.56:10907
  INFO root [0x0000249c] [2018-06-20 19:58:27,336] - 	componentID : 8001
  INFO root [0x0000249c] [2018-06-20 19:58:28,848] - ---- cellapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: D25B4E16028C33EB467B7BE2F3500426. Config: Debug X86. Built: 23:20:13 Jun 15 2018. AppID: 8001. UID: 27688. PID: 10396 ----
  INFO root [0x0000249c] [2018-06-20 19:58:28,849] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x0000249c] [2018-06-20 19:58:28,850] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x0000249c] [2018-06-20 19:58:28,850] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x0000249c] [2018-06-20 19:58:29,022] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x0000249c] [2018-06-20 19:58:29,023] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x0000249c] [2018-06-20 19:58:29,024] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x0000249c] [2018-06-20 19:58:29,031] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x0000249c] [2018-06-20 19:58:29,032] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x0000249c] [2018-06-20 19:58:29,032] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x0000249c] [2018-06-20 19:58:29,033] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x0000249c] [2018-06-20 19:58:29,113] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x0000249c] [2018-06-20 19:58:29,192] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:57) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x0000249c] [2018-06-20 19:58:29,192] - Script::install(): KBExtra is successfully!
  INFO root [0x0000249c] [2018-06-20 19:58:29,213] - TelnetServer server is running on port 50000
  INFO root [0x0000249c] [2018-06-20 19:58:31,400] - ---- cellapp is running ----
 DEBUG root [0x0000249c] [2018-06-20 19:58:31,401] - Components::process(): Request for the process of identity...
  WARN root [0x0000249c] [2018-06-20 19:58:31,713] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20603, errorno=10048
  WARN root [0x0000249c] [2018-06-20 19:58:37,821] - EntityApp::checkTickPeriod: tick took 6420% (6.42 seconds)!
  INFO root [0x0000249c] [2018-06-20 19:58:38,900] - Components::findComponents: find logger(1)...
  INFO root [0x0000249c] [2018-06-20 19:58:38,991] - Components::findComponents: found logger, addr:192.168.31.56:10912
  INFO root [0x0000249c] [2018-06-20 19:58:38,992] - Components::addComponent: reset orderLog, uid=27688!
  INFO root [0x0000249c] [2018-06-20 19:58:38,992] - Components::addComponent[logger], uid=27688, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x0000249c] [2018-06-20 19:58:38,993] - Components::findComponents: find dbmgr(2)...
  INFO root [0x0000249c] [2018-06-20 19:58:39,098] - Components::findComponents: found dbmgr, addr:192.168.31.56:10913
  INFO root [0x0000249c] [2018-06-20 19:58:39,099] - Components::addComponent: reset orderLog, uid=27688!
  INFO root [0x0000249c] [2018-06-20 19:58:39,099] - Components::addComponent[dbmgr], uid=27688, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x0000249c] [2018-06-20 19:58:39,100] - Components::findComponents: found dbmgr, addr:192.168.31.56:10913
  WARN root [0x0000249c] [2018-06-20 19:58:40,100] - EntityApp::checkTickPeriod: tick took 1300% (1.30 seconds)!
  WARN root [0x0000249c] [2018-06-20 19:58:40,101] - Forwarding logs to logger[192.168.31.56:10912/0/0/0]...
 ERROR root [0x0000249c] [2018-06-20 20:00:29,458] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x0000249c] [2018-06-20 20:00:29,466] - The following logs sent to logger failed:
  WARN root [0x0000249c] [2018-06-20 20:00:29,467] - ==> [2018-06-20 20:00:29 455] TCPPacketReceiver::processPendingEvents(192.168.31.56:10912): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x0000249c] [2018-06-20 20:00:29,467] - ==> [2018-06-20 20:00:29 459] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x0000249c] [2018-06-20 20:00:29,554] - TCPPacketReceiver::processPendingEvents(192.168.31.56:10913): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x0000249c] [2018-06-20 20:00:29,559] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x0000249c] [2018-06-20 20:00:29,562] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [0x0000249c] [2018-06-20 20:00:29,564] - Shutdowner::onShutdown: shutting down
  INFO root [0x0000249c] [2018-06-20 20:00:29,565] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [0x0000249c] [2018-06-20 20:00:29,565] - EntityRef::destroyObjPool(): size 0.
 DEBUG root [0x0000249c] [2018-06-20 20:00:29,566] - Witness::destroyObjPool(): size 0.
 DEBUG root [0x0000249c] [2018-06-20 20:00:29,600] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x0000249c] [2018-06-20 20:00:29,600] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x0000249c] [2018-06-20 20:00:29,600] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x0000249c] [2018-06-20 20:00:29,600] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x0000249c] [2018-06-20 20:00:29,601] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x0000249c] [2018-06-20 20:00:29,601] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x0000249c] [2018-06-20 20:00:29,601] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x0000249c] [2018-06-20 20:00:29,602] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x0000249c] [2018-06-20 20:00:29,622] - Script::uninstall(): is successfully!
 DEBUG root [0x0000249c] [2018-06-20 20:00:29,623] - ThreadPool::destroy(): starting size 2.
  WARN root [0x0000249c] [2018-06-20 20:00:29,923] - ThreadPool::destroy(): waiting for thread(2)[0x3531b8,0x353150,], try=1
  INFO root [0x00001a9c] [2018-06-20 22:27:46,446] - 
  INFO root [0x00001a9c] [2018-06-20 22:27:46,447] - -----------------------------------------------------------------------------------------


  INFO root [0x00001a9c] [2018-06-20 22:27:46,448] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x00001a9c] [2018-06-20 22:27:46,449] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x00001a9c] [2018-06-20 22:27:46,449] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x00001a9c] [2018-06-20 22:27:46,459] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:11482 )
  INFO root [0x00001a9c] [2018-06-20 22:27:46,459] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:11482, SOMAXCONN=5.
  INFO root [0x00001a9c] [2018-06-20 22:27:46,459] - server-configs:
  INFO root [0x00001a9c] [2018-06-20 22:27:46,459] - 	gameUpdateHertz : 10
  INFO root [0x00001a9c] [2018-06-20 22:27:46,460] - 	defaultAoIRadius : 80
  INFO root [0x00001a9c] [2018-06-20 22:27:46,460] - 	defaultAoIHysteresisArea : 5
  INFO root [0x00001a9c] [2018-06-20 22:27:46,460] - 	entryScriptFile : kbemain
  INFO root [0x00001a9c] [2018-06-20 22:27:46,460] - 	internalAddr : 192.168.31.56:11482
  INFO root [0x00001a9c] [2018-06-20 22:27:46,460] - 	componentID : 8001
  INFO root [0x00001a9c] [2018-06-20 22:27:47,973] - ---- cellapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: D25B4E16028C33EB467B7BE2F3500426. Config: Debug X86. Built: 23:20:13 Jun 15 2018. AppID: 8001. UID: 24176. PID: 3564 ----
  INFO root [0x00001a9c] [2018-06-20 22:27:47,974] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x00001a9c] [2018-06-20 22:27:47,975] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00001a9c] [2018-06-20 22:27:47,976] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00001a9c] [2018-06-20 22:27:48,132] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001a9c] [2018-06-20 22:27:48,133] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00001a9c] [2018-06-20 22:27:48,134] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00001a9c] [2018-06-20 22:27:48,140] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001a9c] [2018-06-20 22:27:48,141] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00001a9c] [2018-06-20 22:27:48,141] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001a9c] [2018-06-20 22:27:48,142] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001a9c] [2018-06-20 22:27:48,212] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00001a9c] [2018-06-20 22:27:48,267] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:57) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00001a9c] [2018-06-20 22:27:48,267] - Script::install(): KBExtra is successfully!
  INFO root [0x00001a9c] [2018-06-20 22:27:48,282] - TelnetServer server is running on port 50000
  INFO root [0x00001a9c] [2018-06-20 22:27:49,960] - ---- cellapp is running ----
 DEBUG root [0x00001a9c] [2018-06-20 22:27:49,962] - Components::process(): Request for the process of identity...
  WARN root [0x00001a9c] [2018-06-20 22:27:56,093] - EntityApp::checkTickPeriod: tick took 6132% (6.13 seconds)!
  INFO root [0x00001a9c] [2018-06-20 22:27:57,150] - Components::findComponents: find logger(1)...
  INFO root [0x00001a9c] [2018-06-20 22:27:57,215] - Components::findComponents: found logger, addr:192.168.31.56:11487
  INFO root [0x00001a9c] [2018-06-20 22:27:57,215] - Components::addComponent: reset orderLog, uid=24176!
  INFO root [0x00001a9c] [2018-06-20 22:27:57,216] - Components::addComponent[logger], uid=24176, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001a9c] [2018-06-20 22:27:57,218] - Components::findComponents: find dbmgr(2)...
  INFO root [0x00001a9c] [2018-06-20 22:27:57,319] - Components::findComponents: found dbmgr, addr:192.168.31.56:11483
  INFO root [0x00001a9c] [2018-06-20 22:27:57,320] - Components::addComponent: reset orderLog, uid=24176!
  INFO root [0x00001a9c] [2018-06-20 22:27:57,320] - Components::addComponent[dbmgr], uid=24176, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001a9c] [2018-06-20 22:27:57,321] - Components::findComponents: found dbmgr, addr:192.168.31.56:11483
  WARN root [0x00001a9c] [2018-06-20 22:27:58,322] - Forwarding logs to logger[192.168.31.56:11487/0/0/0]...
 ERROR root [0x00001a9c] [2018-06-20 23:41:50,942] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00001a9c] [2018-06-20 23:41:50,945] - The following logs sent to logger failed:
  WARN root [0x00001a9c] [2018-06-20 23:41:50,945] - ==> [2018-06-20 23:41:50 937] TCPPacketReceiver::processPendingEvents(192.168.31.56:11487): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001a9c] [2018-06-20 23:41:50,946] - ==> [2018-06-20 23:41:50 942] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x00001a9c] [2018-06-20 23:41:51,036] - TCPPacketReceiver::processPendingEvents(192.168.31.56:11483): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001a9c] [2018-06-20 23:41:51,042] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x00001a9c] [2018-06-20 23:41:51,067] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [0x00001a9c] [2018-06-20 23:41:51,068] - Shutdowner::onShutdown: shutting down
  INFO root [0x00001a9c] [2018-06-20 23:41:51,068] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [0x00001a9c] [2018-06-20 23:41:51,069] - EntityRef::destroyObjPool(): size 0.
 DEBUG root [0x00001a9c] [2018-06-20 23:41:51,069] - Witness::destroyObjPool(): size 0.
 DEBUG root [0x00001a9c] [2018-06-20 23:41:51,112] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001a9c] [2018-06-20 23:41:51,113] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00001a9c] [2018-06-20 23:41:51,113] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00001a9c] [2018-06-20 23:41:51,113] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001a9c] [2018-06-20 23:41:51,114] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00001a9c] [2018-06-20 23:41:51,114] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001a9c] [2018-06-20 23:41:51,114] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001a9c] [2018-06-20 23:41:51,114] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x00001a9c] [2018-06-20 23:41:51,129] - Script::uninstall(): is successfully!
 DEBUG root [0x00001a9c] [2018-06-20 23:41:51,129] - ThreadPool::destroy(): starting size 2.
